doc/topics/releases/nitrogen.rst

```diff
@@ -61,48 +61,50 @@ State Module Changes
   start/stop the service using the ``--no-block`` flag in the ``systemctl``
   command. On non-systemd minions, a warning will be issued.
 
-- The :py:func:`module.run <salt.states.module.run>` state has dropped its previous
-  syntax with ``m_`` prefix for reserved keywords. Additionally, it allows
-  running several functions in a batch.
+- The :py:func:`module.run <salt.states.module.run>` state has dropped its
+  previous syntax with ``m_`` prefix for reserved keywords. Additionally, it
+  allows running several functions in a batch.
 
-.. note::
-    It is nesessary to explicitly turn on the new behaviour (see below)
+  .. note::
+      It is necessary to explicitly turn on the new behavior (see below)
 
-  Before and after:
+  .. code-block:: yaml
 
-.. code-block:: yaml
+      # Before
+      run_something:
+        module.run:
+          - name: mymodule.something
+          - m_name: 'some name'
+          - kwargs: {
```

Comment:
It's not technically valid YAML, but yes, this does work, and is quite easy to test:

```python
>>> import yaml
>>> import pprint
>>> pprint.pprint(yaml.safe_load('''\
... run_something:
...   module.run:
...     - name: mymodule.something
...     - m_name: 'some name'
...     - kwargs: {
...       first_arg: 'one',
...       second_arg: 'two',
...       do_stuff: 'True'
...     }
... '''))
{'run_something': {'module.run': [{'name': 'mymodule.something'},
                                  {'m_name': 'some name'},
                                  {'kwargs': {'do_stuff': 'True',
                                              'first_arg': 'one',
                                              'second_arg': 'two'}}]}}
>>>
```

This behavior is [already documented](https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html#nested-dictionaries).